package.json

```diff
@@ -10,13 +10,15 @@
   "scripts": {
     "android": "npx react-native run-android",
     "build-prod:ios": "NODE_OPTIONS=--max_old_space_size=8192 npx react-native bundle --dev false --entry-file index.js --bundle-output ios/main.jsbundle --platform ios --assets-dest='./ios'",
-    "build": "yarn install && npx pod-install",
+    "build:ios": "yarn install && npx pod-install",
```

Comment:
Maybe let's call it `install:ios` while we're at it?
It is really confusing to have a `build` target which doesn't, well, _build_ anything.